studio-docs/source/metrics/operation-signatures.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: Operation signatures
+description: How Studio identifies equivalent operations
+---
+
+In your metrics reports, Apollo Studio groups together operations based on the exact set of fields they include, _not_ based on operation _names_. This means that operations with the same name but different sets of fields are displayed separately:
```

Comment:
Worth noting that this is an aspect of the product that we might be changing this year (cc @timbotnik) but that doesn't mean we shouldn't document it.

Reply:
Cool, happy for us to tweak as needed moving forward